proxy/proxy.go

```diff
@@ -39,21 +39,24 @@ var (
 		prometheus.CounterOpts{
 			Name: "pushprox_http_requests_total",
 			Help: "Number of http api requests.",
-		},
-		[]string{"code", "path"},
+		}, []string{"code", "path"},
 	)
 
 	httpProxyCounter = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Name: "pushproxy_proxied_requests_total",
 			Help: "Number of http proxy requests.",
-		},
-		[]string{"code"},
+		}, []string{"code"},
 	)
+	httpPathHistogram = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name: "pushprox_http_path_duration",
```

Comment:
seconds

Reply:
Also, the path is redundant. No need to mention label names in metric names.

Reply:
Fixed